repository/src/main/AndroidManifest.xml

```diff
@@ -1,2 +1,7 @@
-<manifest package="com.melih.repository"/>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.melih.repository">
+
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
Can you move this to `app`'s manifest file ? Permissions will stay in one place then.

Reply:
Sure!